diff -urN aaa/rox-1.3.3/ROX-Filer/src/type.c rox-1.3.3/ROX-Filer/src/type.c
--- aaa/rox-1.3.3/ROX-Filer/src/type.c	Wed Jul 24 19:17:57 2002
+++ rox-1.3.3/ROX-Filer/src/type.c	Fri Jul 26 15:03:51 2002
@@ -1001,6 +1001,10 @@
 	path = "/usr/share/mime/mime-info";
 	if (mc_stat(path, &info) == 0)
 		filer_opendir(path, NULL);
+
+	path = "/usr/X11R6/share/mime-info";
+	  if (mc_stat(path, &info) == 0)
+			    filer_opendir(path, NULL);
 }
 
 static void reread_mime_files(guchar *unused)
@@ -1236,6 +1240,7 @@
 
 	import_file("/usr/share/mime/globs", globs);
 	import_file("/usr/local/share/mime/globs", globs);
+	import_file("/usr/X11R6/share/mime-info/globs", globs);
 
 	tmp = g_strconcat(home_dir, "/.mime/globs", NULL);
 	import_file(tmp, globs);
