diff -ruN rox-1.3.9./ROX-Filer/src/type.c rox-1.3.9/ROX-Filer/src/type.c
--- rox-1.3.9./ROX-Filer/src/type.c	Sun May 18 10:51:51 2003
+++ rox-1.3.9/ROX-Filer/src/type.c	Sun May 18 10:53:10 2003
@@ -1306,7 +1310,7 @@
 	for (i = n_dirs - 1; i >= 0; i--)
 	{
 		char *path;
-		path = g_build_filename(dirs[i], "mime", "globs", NULL);
+		path = g_build_filename(dirs[i], "mime-info", "globs", NULL);
 		import_file(path, globs);
 		g_free(path);
 		g_free(dirs[i]);
