diff -ruN rox-1.3.9./ROX-Filer/src/type.c rox-1.3.9/ROX-Filer/src/type.c
--- rox-1.3.9./ROX-Filer/src/type.c	Sun May 18 10:51:51 2003
+++ rox-1.3.9/ROX-Filer/src/type.c	Sun May 18 10:53:10 2003
@@ -1047,6 +1047,10 @@
 	path = "/usr/share/mime/packages";
 	if (file_exists(path))
 		filer_opendir(path, NULL, NULL);
+
+	path = "/usr/share/mime-info";
+	if (mc_stat(path, &info) == 0)
+		filer_opendir(path, NULL, NULL);
 }
 
 static GList *build_type_edit(Option *none, xmlNode *node, guchar *label)
@@ -1306,7 +1310,7 @@
 	for (i = n_dirs - 1; i >= 0; i--)
 	{
 		char *path;
-		path = g_build_filename(dirs[i], "mime", "globs", NULL);
+		path = g_build_filename(dirs[i], "mime-info", "globs", NULL);
 		import_file(path, globs);
 		g_free(path);
 		g_free(dirs[i]);
