diff -ruN rox-2.1.3./ROX-Filer/src/main.c rox-2.1.3/ROX-Filer/src/main.c
--- rox-2.1.3./ROX-Filer/src/main.c	2004-08-03 14:33:03.000000000 +0200
+++ rox-2.1.3/ROX-Filer/src/main.c	2004-08-03 14:41:03.908216808 +0200
@@ -101,10 +101,10 @@
 		"see the file named COPYING.\n")
 
 #ifdef HAVE_GETOPT_LONG
-#  define USAGE   N_("Try `ROX-Filer/AppRun --help' for more information.\n")
+#  define USAGE   N_("Try `rox --help' for more information.\n")
 #  define SHORT_ONLY_WARNING ""
 #else
-#  define USAGE   N_("Try `ROX-Filer/AppRun -h' for more information.\n")
+#  define USAGE   N_("Try `rox -h' for more information.\n")
 #  define SHORT_ONLY_WARNING	\
 		_("NOTE: Your system does not support long options - \n" \
 		"you must use the short versions instead.\n\n")
@@ -112,7 +112,7 @@
 
 #define BUGS_TO "<rox-devel@lists.sourceforge.net>"
 
-#define HELP N_("Usage: ROX-Filer/AppRun [OPTION]... [FILE]...\n"	\
+#define HELP N_("Usage: rox [OPTION]... [FILE]...\n"	\
        "Open each directory or file listed, or the current working\n"	\
        "directory if no arguments are given.\n\n"			\
        "  -b, --bottom=PANEL	open PAN as a bottom-edge panel\n"	\
